concepticondata/conceptlists/Davies-1985-116.tsv

```diff
@@ -0,0 +1,117 @@
+Davies-1985-116-34	34	tree	906	TREE
+Davies-1985-116-35	35	root	670	ROOT
+Davies-1985-116-36	36	leaf	628	LEAF
+Davies-1985-116-37	37	meat	634	MEAT
+Davies-1985-116-38	38	fat (grease)	323	FAT (ORGANIC SUBSTANCE)
+Davies-1985-116-39	39	egg	744	EGG
+Davies-1985-116-40	40	he eats	1336	EAT
+Davies-1985-116-41	41	he gives it to me	1447	GIVE
+Davies-1985-116-42	42	he sees	1409	SEE
+Davies-1985-116-43	43	he comes	1446	COME
+Davies-1985-116-44	44	louse	1392	LOUSE
+Davies-1985-116-45	45	one	1493	ONE
+Davies-1985-116-46	46	two	1498	TWO
+Davies-1985-116-47	47	his back	1291	BACK
+Davies-1985-116-48	48	his shoulder	1482	SHOULDER
+Davies-1985-116-49	49	his forehead	123	FOREHEAD
+Davies-1985-116-50	50	his thumb	1781	THUMB
+Davies-1985-116-51	51	his leg	1297	LEG
+Davies-1985-116-52	52	his heart	1223	HEART
+Davies-1985-116-53	53	his bone	1394	BONE
+Davies-1985-116-54	54	his blood	946	BLOOD
+Davies-1985-116-55	55	wing	1257	WING
+Davies-1985-116-56	56	tail	1220	TAIL
+Davies-1985-116-57	57	his father	1217	FATHER
+Davies-1985-116-58	58	his mother	1216	MOTHER
+Davies-1985-116-59	59	brother (older, of man)	1262	BROTHER
+Davies-1985-116-60	60	sister	1263	SISTER
+Davies-1985-116-61	61	name	1405	NAME
+Davies-1985-116-62	62	pig	1337	PIG
+Davies-1985-116-63	63	cassowary	466	CASSOWARY
+Davies-1985-116-64	64	wallaby	165	WALLABY
+Davies-1985-116-65	65	flying fox	235	FLYING FOX
+Davies-1985-116-66	66	rat	1490	RAT
+Davies-1985-116-67	67	frog	503	FROG
+Davies-1985-116-68	68	snake	730	SNAKE
+Davies-1985-116-69	69	taro	351	TARO
+Davies-1985-116-70	70	sugarcane	146	SUGAR CANE
+Davies-1985-116-71	71	yam	410	YAM
+Davies-1985-116-72	72	banana	868	BANANA
+Davies-1985-116-73	73	sweet potato	159	SWEET POTATO
+Davies-1985-116-74	74	axe (stone)	4	AXE
+Davies-1985-116-75	75	net bag	293	NETBAG
+Davies-1985-116-76	76	house	1252	HOUSE
+Davies-1985-116-77	77	earth	1228	EARTH (SOIL)
+Davies-1985-116-78	78	mountain	639	MOUNTAIN
+Davies-1985-116-79	79	wind	960	WIND
+Davies-1985-116-80	80	night	1233	NIGHT
+Davies-1985-116-81	81	white	1335	WHITE
+Davies-1985-116-82	82	black	163	BLACK
+Davies-1985-116-83	83	red	156	RED
+Davies-1985-116-84	84	green	1425	GREEN
+Davies-1985-116-85	85	good	1035	GOOD
+Davies-1985-116-86	86	long	1203	LONG
+Davies-1985-116-87	87	short	1645	SHORT
+Davies-1985-116-88	88	heavy	1210	HEAVY
+Davies-1985-116-89	89	cold	1287	COLD
+Davies-1985-116-90	90	warm, hot	2272	HOT OR WARM
+Davies-1985-116-91	91	old	1229	OLD
+Davies-1985-116-92	92	new	1231	NEW
+Davies-1985-116-93	93	many	1198	MANY
+Davies-1985-116-94	94	what?	1236	WHAT
+Davies-1985-116-95	95	who?	1235	WHO
+Davies-1985-116-96	96	wet	1726	WET
+Davies-1985-116-97	97	full	1429	FULL
+Davies-1985-116-98	98	not	1240	NOT
+Davies-1985-116-99	99	three	492	THREE
+Davies-1985-116-100	100	four	1500	FOUR
+Davies-1985-116-101	101	no	1269	NO
+Davies-1985-116-102	102	he says	1458	SAY
+Davies-1985-116-103	103	he hears	1408	HEAR
+Davies-1985-116-104	104	he dies	1494	DIE
+Davies-1985-116-105	105	it burns	2102	BURN
```

Comment:
use 1428 BURNING, as this is obviously intransitive

Reply:
done